static/css/restyle.less

```diff
@@ -0,0 +1,853 @@
+    }
+
+    li {
+        border-style: none;
+    }
+}
+
+.island .pager {
+    display: none;
+}
+#featured-extensions {
+    padding-top: 0;
+}
+.sidebar-breakout h3, h2 {
+    font: inherit;
+    font-size: 1.25rem;
+}
+
+#footer {
+    background-color: #fff;
+    background-image: linear-gradient(to bottom,
+      #fff 0,
+      #fff 123px,
+      rgba(12, 153, 213, 1) 123px,
+      rgba(12, 153, 213, 1) 200px,
+      #FFF 200px,
+      rgba(12, 153, 213, 0.0) 400px);
+    font-family: inherit;
+    font-style: normal;
+}
+#lang_form {
+     margin-top: 12px;
+}
+
+/* --------------------*/
+/* Add-on Details Page */
+/* --------------------*/
+#page #breadcrumbs {
+    display: none;
+}
+.addon-details .primary,
+.addon-details .secondary {
+    font-family: "Open Sans",X-LocaleSpecific,sans-serif !important;
+}
+
+#addon h1 {
+    color: #fff;
+}
+.version-number {
+    color: #ccc;
+    font-size: .5em;
+}
+/* first block of details - icon, name, description ... */
+.addon-details .primary:first-of-type{
+    padding-bottom: 1em; /* */
+    min-height: 257px; /* make sure it is at least as high as the background extends down */
+}
+.addon-details .secondary.addon-vitals {
+    position: relative;
+    z-index: 1;
+}
+
+#addon-description.prose {
+     font-family: 'Open Sans';
+}
+
+/* DevHub */
+#devhub-promo h2 strong {
+    font-family: 'Open Sans';
+}
+#masthead h1 {
+    margin-top: -5px;
+}
+/* Re-colors links on top of details page to white */
+#addon a, .author a,
+.addon-vitals .widgets .widget { 
+    color: #FFFFFF; 
+}
+#addon-summary a,
+.notice.c.author a {
+    color: @primary-link;
+}
+
+.install-button > .button.add.installer {
+    background: #57BD35;
+    box-shadow: 0 2px 0 0 rgba(0,0,0,0.10), inset 0 -2px 0 0 rgba(56,136,30,0.50);
+    border-radius: 3px;
+    font-size: 14px;
+    color: #FFFFFF;
+    line-height: 16px;
+}
+
+.install-button > .button.add.installer:hover {
+    background: #90E07F;
+}
+.install-button > .button.add.installer:active {
+    background: #40A624;
+    box-shadow: none;
+}
+
+/* Notice - underneath description (asking for contribution or  )*/
+.notice {
+    border-radius: 0;
+    box-shadow: none;
+    border: 0;
+    border-top: 1px solid #CCD6E3;
+    background: 0;
+}
+
+/* move elements in front of the background hack */
+.addon-details .primary .notice,
+.addon-details .secondary.addon-vitals {
+    position: relative;
+    z-index: 1;
+}
+
+/* Notice - underneath description (asking for contribution &/or showing dependencies )*/
+.addon-details .primary .notice {
+    border-radius: 0;
+    box-shadow: none;
+    border: 0;
+    border-top: 1px solid rgba(255,255,255,0.3);
+    background: 0;
+}
+
+/* modify dependency to show "add to firefox" button and hide overlay */
+/* make hovercard hide discription and always show "add to firefox" */
+.addon-details .primary .notice.dependencies {
+    height: 130px;
+}
+.addon-details .primary .notice .hovercard.addon:hover,
```

Comment:
Since it's LESS, could you do:

```
.addon-details .primary .notice {
    .hovercard.addon:hover {
        [...]
    }
}
```
to reduce the noise?